statsmodels/stats/diagnostic.py

```diff
@@ -845,12 +845,16 @@ def het_breuschpagan(resid, exog_het):
     exog_het : array_like
         This contains variables suspected of being related to
         heteroscedasticity in resid.
+    robust : bool, default False
+        Flag indicating whether to use the Koenker version of the
+        test (default) which assumes independent and identically distributed
+        error terms, or the original Breusch–Pagan version.
```

Comment:
Maybe clarify

, or the original Breusch-Pagan version which assumes residuals are normally distributed.